mitodl/ocw-hugo-projects
# add basic ocw-www config

This PR adds the basic Hugo config for the `ocw-www` website as well as an empty `ocw-studio` config and a basic `.gitignore` to get us started.

To test this PR:
 - Clone https://github.com/mitodl/ocw-hugo-themes
 - From `ocw-hugo-themes` directory, run `yarn install && npm run start:site:webpack`
 - Clone https://github.com/mitodl/ocw-www
 - From `ocw-www` directory, run:
 ```
hugo server --config /path/to/ocw-hugo-projects/ocw-www/config.yaml --themesDir /path/to/ocw-hugo-themes/
 ```

The OCW home page and search page should be spun up at http://localhost:1313
